docs/Examples/watch-only-wallet.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const {HDPrivateKey, Network, Mnemonic} = require('bcoin');
```

Comment:
The other examples are using `require('../..')` for `bcoin` so that these can be run here.

Reply:
👍 I've got a lot to do in this PR...